src/pages/schedule-page/read-only-tab/read-only-toolbar.tsx

```diff
@@ -44,7 +49,21 @@ export function ReadOnlyToolbar({ openEdit }: ViewOnlyToolbarOptions): JSX.Eleme
 
     const isRevisionEditDisable = revision === "actual" ? false : !isFuture;
     setEditDisable(isRevisionEditDisable || isCorrupted);
-  }, [year, month, revision, isCorrupted]);
+
+    if (
+      AlgorithmErrorCode.AlwaysAtLeastOneNurse in scheduleErrors ||
+      AlgorithmErrorCode.WorkerNumberDuringDay in scheduleErrors ||
+      AlgorithmErrorCode.WorkerNumberDuringNight in scheduleErrors ||
+      AlgorithmErrorCode.DissalowedShiftSequence in scheduleErrors ||
+      AlgorithmErrorCode.LackingLongBreak in scheduleErrors ||
+      AlgorithmErrorCode.WorkerUnderTime in scheduleErrors ||
+      AlgorithmErrorCode.WorkerOvertime in scheduleErrors
```

Comment:
`Object.values(AlgorithmErrorCode).some(errorCode =>  errorCode in scheduleErrors)`

Reply:
Dzięki, poprawione